### PR TITLE
fix: pass arguments to substra-tools via a file

### PR DESCRIPTION
**Pull request by AurelienGasser**
Companion PRs:

- https://github.com/Substra/substra-tools/pull/55 (main PR)
- https://github.com/Substra/substra/pull/279 (this PR)
- https://github.com/Substra/substra-backend/pull/473

---

This fixes a bug when the substra-tools command line argument list is too long.

### CODE REVIEW

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by AurelienGasser** [comment]
:warning: Do not merge until substra-tools and been released

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by AurelienGasser** [comment]
This is ready for review. `make test` passes locally :heavy_check_mark: Unit tests won't pass in the CI until substra-tools is released.

File: substra/sdk/backends/local/compute/spawner/subprocess.py [hunk 1]
```diff
@@ -49,38 +50,12 @@ def _get_entrypoint_from_dockerfile(tmpdir):
...
-    # replace paths with spaces that were modified
-    # Not using regex as shlex replaced '\' with '\\' automatically. Hence '\\\\' which was represented as '\\' is now
-    # '\\\\'
-    command_args = [command_arg.replace("\\\\", " ") for command_arg in command_args]
-    # put current python interpreter and script to launch before script command
-    py_command = [sys.executable, str(tmpdir / script_name), "--method-name", str(method_name)]
-    py_command.extend(command_args)
-
-    return py_command
+def _get_command_args(
```
Comment (Esadruhn) [inline substra/sdk/backends/local/compute/spawner/subprocess.py:53]: Thanks for the cleaning, this is much simpler

**Review by Esadruhn — APPROVED** [review]
Great, thanks for the PR!

**Pull request merged**